d2layouts/d2sequence/sequence_diagram.go

```diff
@@ -386,15 +386,24 @@ func (sd *sequenceDiagram) addLifelineEdges() {
 		}
 		actorLifelineEnd := actor.Center()
 		actorLifelineEnd.Y = endY
+		style := d2graph.Style{
+			StrokeDash:  &d2graph.Scalar{Value: fmt.Sprintf("%d", LIFELINE_STROKE_DASH)},
+			StrokeWidth: &d2graph.Scalar{Value: fmt.Sprintf("%d", LIFELINE_STROKE_WIDTH)},
+		}
+		if actor.Attributes.Style.StrokeDash != nil {
+			style.StrokeDash = actor.Attributes.Style.StrokeDash
```

Comment:
I think it might be safer to make a copy of the scalar vs having both point to the same struct, but not sure that it matters here

Reply:
let me address this quickly!